quickstarts/dotnet/csharp/config.yml

```diff
@@ -0,0 +1,55 @@
+
+slug: csharp
+description: |
+  ## Comprehensive monitoring quickstart for C#
+  With our C# integration, you can find and address performance bottlenecks while operating in a live development environment. Monitor your app via our dashboard quickstarts and ensure you're delivering the best user experience.
+  
+  ## Why monitor C#?
+  - Focus on the issues that affect your critical business transactions
+  - Resolve problems quickly
+  - Gain knowledge on your system's behavior in advance
+  - Learn the whole story of performance errors, from beginning to end
+  - Use traces to connect latency issues to errors, getting you to the root cause as quickly as possible
+  - See the filename and line number of the C# stack trace so you never have to guess. To reduce noise, logically filter and aggregate C# exceptions.
+  - Scalable error monitoring without affecting production flow
+  - Maintain low latency with fast throughput
+  - Get a bird's eye view of your C# applications using New Relic’s dashboard and get meaningful data about C# errors
+  
+  ## Get end-to-end visibility into your C# operation terrain and all the dependencies
+  
+  Automatically discover operation topology and interdependencies, relate and trace crucial business deals grounded on product operation address feature and prioritize the end-to-end business deals performance, not just the health of the usage and structure lumps.
+
+  ## What’s included in this quickstart?
+  New Relic's C# monitoring quickstart boasts instant full-stack observability out-of-the-box:
+  
+  - Dashboards (Throughput, Error rate, Logs, Web transaction time, Apdex Score and more)
+  - Monitor scripts, functions, and web transactions.
+  - High-value alerts
+  - Code-related insights about your application’s health and status. 
+  - Proactive alerts that inform developers about the status of their applications.
+
+summary: |
+  Monitor your C# applications by tracking performance issues with a full end-to-end view of distributed traces.
+icon: logo.svg
+level: New Relic
+authors:
+  - New Relic
+title: C#
+documentation:
+  - name: C# installation docs
+    description: |
+      Quickly monitor performance issues and view full end-to-end distributed trace of your .NET applications built with C#.
+    url: https://docs.newrelic.com/docs/agents/net-agent/getting-started/compatibility-requirements-net-framework-agent
```

Comment:
Hi @akristen , request you to kindly provide us the right docs url, so that we can update this PR with it. 😃